core/store/migrate/migrations/0063_add_job_proposal_timestamp.sql

```diff
@@ -0,0 +1,12 @@
+-- +goose Up
+ALTER TABLE job_proposals
+    ADD COLUMN proposed_at TIMESTAMP WITH TIME ZONE;
```

Comment:
You could just declare it as `NOT NULL DEFAULT now()`, it'll take care of existing entries

Reply:
Though yeah it will differ from created_at

Reply:
We want it to take the time of `created_at` because that was being used for proposals initially.

Reply:
> In order to migrate the data, you can copy the created_at timestamp of the proposal into the new proposed_at field

This is from the ticket. @jkongie what do you think about that? Should we keep it as it is or use the current timestamp by default?

Reply:
Or we can use `NOT NULL DEFAULT now()` option but then align with created at value anyway.